server/controllers/inventory/depots/index.js

```diff
@@ -116,6 +116,14 @@ function list(req, res, next) {
     options.user_id = req.session.user.id;
   }
 
+  if (options.exception && Array.isArray(options.exception)) {
```

Comment:
Do you really need this?  What happens if you just pass `options.exception to `db.convert()`?  Like so:

```js
db.convert(options, ['exception']);
```
From my reading of [the code](https://github.com/IMA-WorldHealth/bhima/blob/master/server/lib/db/index.js#L209) it should do all this for you automagically.

Reply:
Yes, for pepkits for example they asked me if it is possible to hide some depots which are not in a given health zone. Having an array of uuids will help us to do that

Reply:
So, why does db.convert() not handle arrays?  Is there a bug in it?  [This code](https://github.com/IMA-WorldHealth/bhima/blob/master/server/lib/db/index.js#L221)
```js
      if (prop && _.isArray(prop)) {
        // Every item should be converted to binary
        data[key] = data[key].map(this.bid);
      }
```
should be doing exactly what you are doing.  Is there a reason by db.convert() is failing here?

Reply:
It works